client/Components/Header/Header.jsx

```diff
@@ -1,33 +1,38 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { NavLink } from 'react-router-dom';
+import { Link, NavLink } from 'react-router-dom';
 import { ROUTE_PATH } from '../../commons/constants'
 import dogLogo from '../../assets/img/logo-shoppadog.png'
+import dogCrate from '../../assets/img/logo-dogcrate.png'
 import './header.scss'
 
 class Header extends Component {
   render() {
     const { user } = this.props
     return (
       <header>
-        
-        <h1>SHOPPA DOG!</h1>
-        <img id="header-logo" src={dogLogo}/>
-        {user.name && (
-          <>
-            <span>Hi, {user.name}!</span>
-            <NavLink to={ ROUTE_PATH.USER_PROFILE }>My Account</NavLink>
-            <NavLink to={ ROUTE_PATH.USER_ORDERS }>Order History</NavLink>
-            <a href="/api/auth/logout">Logout</a>
-          </>
-        )
-        }
-        {!user.name && (
-          <>
-            <a href="/api/auth/google">Login with GOOGLE+</a>
-          </>
-        )
-        }
+        <div id="logo">
```

Comment:
code climate is mad. we can resolve this by increasing/disabling the line limit... but we can also break this out to smaller components... like `NavBar`, etc..